src/libraries/System.Runtime/tests/TrimmingTests/System.Runtime.TrimmingTests.proj

```diff
@@ -16,6 +16,7 @@
       <DisabledFeatureSwitches>System.Globalization.Invariant</DisabledFeatureSwitches>
     </TestConsoleAppSourceFiles>
     <TestConsoleAppSourceFiles Include="InvariantGlobalizationTrue.cs">
+      <DisabledFeatureSwitches>System.Globalization.Hybrid</DisabledFeatureSwitches>
```

Comment:
Does trimming work fine when Hybrid is enabled ?

Reply:
We don't trim when Hybrid is enabled. Trimming should work only when Invariant is on. When Invariant is on then Hybrid gets automatically overwritten to be off. That's why we cannot test Invariant = true and Hybrid = true because it does not make sense.